packages/main-ui/src/pages/politicians/email_confirmation_popup.rs

```diff
@@ -56,38 +53,35 @@ pub fn EmailConfirmationPopup(
                                         })
                                         .with_id("contact_us_popup")
                                         .with_title(tr.contact_us);
-                                    },
-                                " {tr.here}" 
+                                },
+                                " {tr.here}"
                             }
-                        },
-                        span { class: "font-normal", "{tr.explanation_confirm_email2}" },
+                        }
+                        span { class: "font-normal", "{tr.explanation_confirm_email2}" }
                     }
                 }
                 div { class: "flex w-full",
                     button {
                         class: "w-full h-[57px] text-[{theme.primary05}] bg-[{theme.primary03}] text-[18px] font-extrabold leading-[24px] rounded-[12px] disabled:opacity-50",
-                        onclick: move |_| { 
+                        onclick: move |_| {
                             tracing::debug!("confirm button clicked");
-                            // TODO: check email verification
-                            popup.
-                                open(
-                                    rsx! {
-                                        ProclaimPopup {
-                                            class: "w-[450px]",
-                                            name: name.clone(),
-                                            party: party.clone(),
-                                            stance: stance.clone(),
-                                            lang: lang.clone(),
-                                        }
-                                    })
-                                    .with_id("proclaim_popup")
-                                    .with_title(tr.proclaim);
-
-                         },
+                            popup
+                                .open(rsx! {
+                                    ProclaimPopup {
+                                        class: "w-[450px]",
+                                        name: name.clone(),
+                                        party: party.clone(),
+                                        stance: stance.clone(),
+                                        lang: lang.clone(),
```

Comment:
```suggestion
                                        lang,
```
lang 은 Copy 가 동작하기 때문에 clone 할 필요 없습니다.